how-to-docs/drone-how-to.md

```diff
@@ -315,70 +315,74 @@ Drone will automatically execute that step when a new pull request is raised.
 
 [Read more about Drone conditions](http://docs.drone.io/conditional-steps/).
 
-## Deploying to DSP
+## Deploying to ACP
 
-> Please note that this section assumes you have a separate repository containing your kube files as explained [here](https://github.com/UKHomeOffice/application-container-platform/blob/master/developer-docs/platform_introduction.md#define-a-deployment-for-your-application).
+Add a deployment script with the following:
+ 
+```bash
+#!/bin/bash
+export KUBE_NAMESPACE=<dev-induction>
+export KUBE_SERVER=${KUBE_SERVER}
+export KUBE_TOKEN=${KUBE_TOKEN}
+  
+kd --insecure-skip-tls-verify \
+    -f deployment.yaml \
+    -f service.yaml \
+    -f ingress.yaml
+```
 
-You can clone your kube repo as part of your pipeline with:
+> Please note that this is only an example script and it will need to be changed to fit your particular application's needs.
 
-```yaml
-predeploy_to_uat:
-  image: plugins/git
-  commands:
-    - git clone https://${GITHUB_TOKEN}:x-oauth-basic@github.com/UKHomeOffice/<your_repo>.git
-  when:
-    environment: uat
-    event: deployment
+If you deployed this now you would likely receive an error similar to this:
+
+```
+error: You must be logged in to the server (the server has asked for the client to provide credentials)
 ```
 
-from now on, the repository is part of the workspace and is ready to be accessed by other steps in the pipeline.
+[kd](https://github.com/UKHomeOffice/kd) uses 3 environment variables to deploy:
```

Comment:
Minor: it's not super clear that the previous error is because `kd` needs these environment variables (when I first read this I thought you had skipped ahead to explaining something else). Maybe something as simple as starting with `This error is because …`.